browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -349,6 +354,9 @@ var NotebookbarAccessibility = function() {
 					if (this.activeTabPointers.id !== element.id) {
 						this.setupAcceleratorsForCurrentTab(element.id);
 					}
+					if (!document.body.classList.contains('activate-info-boxes')) {
+						this.resetState();
+					}
```

Comment:
We shouldn't need this 'click' function entirely, including the "setupAcceleratorsForCurrentTab" part. Info boxes should be cleaned on blur and state should be reset at the same time.